src/components/toolbar/Toolbar.tsx

```diff
@@ -91,25 +93,27 @@ const Toolbar = (props: ToolbarProps) => {
           }
         />
       </Innhold>
-      <Row className="flex ml-0 mr-0 items-center">
-        <Column xs="1">
-          <VelgBoks
-            className="toolbar__velgBoks"
-            aria-label={tekster.selectAll}
-            label="&nbsp;"
-            checked={props.alleMarkert}
-            onChange={(event) => {
-              props.checkAllHandler(event.target.checked);
+      {!toggles.isAkselOversiktEnabled && (
```

Comment:
Viser bare `<Sorteringsrad>` og checkbox for å velge rader for gammel tabell (Aksel-tabell har dette innebygd).